src/test/integrationTest/authRoute.test.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable camelcase */
+import chai, { expect, should } from 'chai';
+import chaiHTTP from 'chai-http';
+import server from '../..';
+
+should();
+chai.use(chaiHTTP);
+const route = '/api/v1/auth/signup';
+const generateNumber = () => `${Math.round(Math.random() * 1000)}${Date.now()}`;
+const validUser = {
+  email: `john${generateNumber()}@doe.com`,
+  password: 'this1sN0TmyPassword'
+};
+
+const shortPassword = {
+  email: `john${generateNumber()}@doe.com`,
+  password: 'short'
+};
+
+const nonAlphaNumPassword = {
+  email: `john${generateNumber()}@doe.com`,
+  password: 'isThisInvalid!? :-)'
+};
+
+const invalidEmail = {
+  email: 'invalidEmail',
+  password: 'noNeedForPasswordAnyMore'
+};
+
+const nullEmail = {
+  password: 'noNeedForPasswordAnyMore'
+};
+
+const nullPassword = {
+  email: `john${generateNumber()}@doe.com`
+};
+
```

Comment:
Instead of creating different request body in here, you can always seed the database. However, seeding the database for this test is not necessary. Since this is an integration test, just create different if blocks that would test each of your validation. Also, try to use `expect` instead of `should` for your assertions.

Reply:
I've resolved it. Thanks.